ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -241,16 +265,22 @@ public boolean requestFocus(int direction, Rect previouslyFocusedRect) {
     // is a controlled component, which means its focus is controlled by JS, with two exceptions:
     // autofocus when it's attached to the window, and responding to accessibility events. In both
     // of these cases, we call requestFocusInternal() directly.
+    // Always return true if we are already focused. This is used by android in certain places,
+    // such as text selection.
     return isFocused();
   }
 
   private boolean requestFocusInternal() {
     setFocusableInTouchMode(true);
-    // We must explicitly call this method on the super class; if we call requestFocus() without
-    // any arguments, it will call into the overridden requestFocus(int, Rect) above, which no-ops.
     boolean focused = super.requestFocus(View.FOCUS_DOWN, null);
-    if (getShowSoftInputOnFocus()) {
+    if (getShowSoftInputOnFocus() && !isTVDevice()) {
```

Comment:
I believe this is the cause of #129 .... @vdmtrv would it be ok to revert this?  Why was this added for TV devices?